metrics/web/test/base/presentation/widgets/dropdown_body_test.dart

```diff
@@ -59,6 +66,23 @@ void main() {
       },
     );
 
+    testWidgets(
+      "applies the given child builder",
+      (tester) async {
```

Comment:
It seems like we should test that the widget displays a widget built with the given builder. Instead, this test case checks that the testbed applies the builder to the widget under tests.